### ISSUE TITLE
Network Settings > Local Network Access is missing from the settings menu

### ISSUE THREAD

**Issue by ToeKneeFan** [mozilla-mobile/mozilla-vpn-client#7653]
**Describe the bug**
Network Settings > Local Network Access is missing from the settings menu. There does not appear to be a way to activate the feature.

**VPN version:**
  v2.15.3

**Affected Platforms:**
  Ubuntu 22.04 LTS

**Prerequisites:**
  1. Install Mozilla VPN through command line from the official PPA.

**Steps to reproduce:**
   1. Navigate to settings.
   2. Look at the available options.

**Expected result:**
  Expect to see Network Settings, under which is the option to activate Local Network Access.

**Actual result:**
  No Network Settings option is available in the settings menu.

**Additional notes:**
![image](https://github.com/mozilla-mobile/mozilla-vpn-client/assets/44327479/218d3672-cf88-4d10-baed-f4ec712b3ab4)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-5308)

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Hello,

Thank you for filling this ticket.  The visibility of the Local Network Access option was removed from the Settings menu in 2.13. For more details, please see https://mozilla-hub.atlassian.net/browse/VPN-3232 ( https://mozilla-hub.atlassian.net/browse/VPN-3232|smart-link )

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

About the Atlassion page, Santiago Andrigo might be able to give additional details. 

Regarding the Local Network Access feature, it is enabled by default, so you should be able to connect to local network resources while the VPN is ON. Hope this helps!

**Comment by ToeKneeFan** [comment]
Thank you, that makes sense. In my case, it seems that bridged connections are still blocked by the VPN, but that is out of scope of the issue above, so I will close.
